### PR TITLE
This Month in Rust OSDev: September 2022 

### PR DESCRIPTION
**Pull request by phil-opp**
This is a draft for the upcoming _"This Month in Rust OSDev: September 2022"_ post. Please add your content to this draft by creating a pull request against the `next` branch. All projects related to OS development in Rust are valid here, they don't need to be associated with the `rust-osdev` organization.

The post will be published in the first week of October.

### CODE REVIEW

**Comment by phil-opp** [comment]
It's time to prepare the new issue. Please submit your content in the next few days. All projects are welcome!

I created basic changelogs for the `bootloader`, `uefi`, and `xhci` crates. Feel free to improve these sections if you like.

cc @rust-osdev/all

**Comment by berkus** [comment]
Duh, still fighting with code (mis-)generation, will skip updates this month.

File: content/this-month/2022-09/index.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+We fixed the issues in the following way:
+
+- [Limit BIOS bootloader's `max_phys_addr` to 4 GiB](https://github.com/rust-osdev/bootloader/pull/260)
+- [Only perform a single TLB flush after identity mapping](https://github.com/rust-osdev/bootloader/pull/265)
+- [fix `get_free_address` for large sizes (0.10)](https://github.com/rust-osdev/bootloader/pull/263) <span class="gray">(published as `0.10.13`)
+- [allow allocating more than one level 4 entry (0.9)](https://github.com/rust-osdev/bootloader/pull/264)  <span class="gray">(published as `0.9.23`)
+
+Thanks to [@hawkw](https://github.com/hawkw) and [@Freax13](https://github.com/Freax13) for these contributions!
+
+We also brought the [upcoming `v0.11` release](https://github.com/rust-osdev/bootloader/pull/232) even closed to the finish line. Thanks to an extensive review of [@Freax13](https://github.com/Freax13), we found and fixed multiple remaining issues. Also thanks to [@asensio-project](https://github.com/asensio-project) and [@TheBotlyNoob](https://github.com/TheBotlyNoob) for testing the new version and reporting bugs! Some notable changes are:
```
Comment (Freax13) [inline content/this-month/2022-09/index.md:59]:
```suggestion
We also brought the [upcoming `v0.11` release](https://github.com/rust-osdev/bootloader/pull/232) even closer to the finish line. Thanks to an extensive review of [@Freax13](https://github.com/Freax13), we found and fixed multiple remaining issues. Also thanks to [@asensio-project](https://github.com/asensio-project) and [@TheBotlyNoob](https://github.com/TheBotlyNoob) for testing the new version and reporting bugs! Some notable changes are:
```

Comment (phil-opp) [reply content/this-month/2022-09/index.md:59]: Thanks!

**Comment by phil-opp** [comment]
Published at <https://rust-osdev.com/this-month/2022-09/>. Thanks everyone!

**Pull request merged**